.env.example

```
@@ -8,4 +8,10 @@ ETHERSCAN_API_KEY=your_etherscan_api_key_here
 VRF_SUBSCRIPTION_ID=your_subscription_id_here
 
 # Deployed Raffle contract address
-RAFFLE_CONTRACT_ADDRESS=your_raffle_address_here
\ No newline at end of file
+RAFFLE_CONTRACT_ADDRESS=your_raffle_address_here
+
+# Chainlink Automation configuration
+AUTOMATION_UPKEEP_ID=your_upkeep_id_here
+UPKEEP_NAME=Raffle Auto Draw
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)